src/views/enroll-factors/BarcodeView.js

```diff
@@ -44,9 +44,25 @@ define([
     events: {
       'click [data-type="manual-setup"]': function (e) {
         e.preventDefault();
-        var url = RouterUtil.createActivateFactorUrl(this.model.get('__provider__'),
+        const url = RouterUtil.createActivateFactorUrl(this.model.get('__provider__'),
           this.model.get('__factorType__'), 'manual');
-        this.options.appState.trigger('navigate', url);
+        if (this.model.get('__factorType__') === 'push') {
```

Comment:
for future, wonder if this same problem happens with other factors... just food for thought, nothing to change or add to this PR

Reply:
here this is to check for push or totp. I  dont think there is polling in other factors,

Reply:
right, so does the same problem happen for totp? If so, we could probably file a jira for later...

Reply:
I believe we don't have the same problem for totp because we need only verify by passing in a totp in the browser client and sending that request.

We only have to poll for OVP because the request doesn't come from the browser client. Instead, it's from a separate mobile client, but we still need to advance the browser view, so we have to poll the server to check whether it received that out-of-band verification.